### ISSUE TITLE
Hammers

### ISSUE THREAD

**Issue by DarkScorn** [Magneticraft-Team/Magneticraft#164]
Combining two hammers causes unlimited hammers to be duplicated, after combining, for some reason hammers in the grid does not disappear, I have a chest full of stone hammers now. This bug began after I updated the mod to 2.7.0

**Comment by cout970** [comment]
Oh, I made a change to the hammers so you could craft places without losing the hammer, but it seems to also affect the repair recipe. I need to remove one of the two features, repairing or crafting plates.

Alternatively I could add a different early game plate automation, but I don't have any good idea for that.

**Comment by DarkScorn** [comment]
Well, before that I was wondering why combining two hammers take 1 durability of one of them instead of removing them.
 And I want to make something clear, how does this repair works? I did not see that in-game, although I saw the change about plate recipes made with material blocks, that was a good idea to craft mass plates, and still beneficial even though it only saves 1 ingot. Also gives more reason to make the hydraulic press.
